tests/acceptance/features/apiSpacesDavOperation/moveByFileId.feature

```diff
@@ -221,11 +217,83 @@ Feature: moving/renaming file using file id
     When user "Alice" renames a file "folder/sub-folder/textfile.txt" into "/renamed.txt" inside space "Personal" using file-id path "<dav-path>"
     Then the HTTP status code should be "201"
-    And for user "Alice" the space "Personal" should contain these entries:
+    And for user "Alice" the space "Personal" should contain these files:
       | renamed.txt |
     But for user "Alice" folder "folder/sub-folder" of the space "Personal" should not contain these files:
       | textfile.txt |
     Examples:
       | dav-path                          |
       | /remote.php/dav/spaces/<<FILEID>> |
       | /dav/spaces/<<FILEID>>            |
+
+
+  Scenario Outline: move a file from project to personal space
+    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "project-space" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "project-space" with content "some data" to "textfile.txt"
+    And we save it into "FILEID"
+    When user "Alice" moves a file "/textfile.txt" into "/" inside space "Personal" using file-id path "<dav-path>"
+    Then the HTTP status code should be "502"
+    And the value of the item "/d:error/s:message" in the response about user "Alice" should be "move:error: not supported: cannot move across spaces"
+    And for user "Alice" folder "/" of the space "project-space" should contain these files:
+      | textfile.txt |
+    But for user "Alice" folder "/" of the space "Personal" should not contain these files:
+      | textfile.txt |
+    Examples:
+      | dav-path                          |
+      | /remote.php/dav/spaces/<<FILEID>> |
+      | /dav/spaces/<<FILEID>>            |
+
+
+  Scenario Outline: move a file from personal to project space
+    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "project-space" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "Personal" with content "some data" to "textfile.txt"
+    And we save it into "FILEID"
+    When user "Alice" moves a file "/textfile.txt" into "/" inside space "project-space" using file-id path "<dav-path>"
+    Then the HTTP status code should be "502"
+    And the value of the item "/d:error/s:message" in the response about user "Alice" should be "move:error: not supported: cannot move across spaces"
+    And for user "Alice" folder "/" of the space "Personal" should contain these files:
+      | textfile.txt |
+    But for user "Alice" folder "/" of the space "project-space" should not contain these files:
+      | textfile.txt |
+    Examples:
+      | dav-path                          |
+      | /remote.php/dav/spaces/<<FILEID>> |
+      | /dav/spaces/<<FILEID>>            |
+
+
+  Scenario Outline: rename a file and move a file from project space to personal space
```

Comment:
```suggestion
  Scenario Outline: move a file to different name from project space to personal space
```

Reply:
There are also other scenario which are merged and their steps are similar . I just tries to describe scenario outline with them. If we changed this also should we have to changed other too having similar scenario including the merged one also?